examples/ex14.cpp

```diff
@@ -30,41 +30,96 @@
 //               We recommend viewing examples 1 and 9 before viewing this
```

Comment:
I have modified this example for quick testing.  Currently there are four initial conditions (see "-i") that are run for the PA operator and the full operator.  Then, the output is compared:  
``` error = ||y-yfull ||_inf ```
The error should be ``O(epsilon)`` as the operators are equivalent. 

Currently a bug remains in the restriction-tranpose operator, but that will be squished soon. Once all the bugs are gone, all of the testing code will be cleaned up.